### PR TITLE
adding Michigan Medicare Exclusions crawler

### PR DESCRIPTION
**Pull request by bgmello**
There are multiple sanction dates in the data. I used only the first one because I'm not sure what are the others.

### CODE REVIEW

File: datasets/us/mi/med_exclusions/crawler.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+from rigour.mime.types import XLSX
+from openpyxl import load_workbook
+
+from zavod import Context, helpers as h
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    if not row.get("entity_name"):
+        return
```
Comment (jbothma) [inline datasets/us/mi/med_exclusions/crawler.py:11]: it looks like sometimes there's a first name but not a legal entity

File: datasets/us/mi/med_exclusions/crawler.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    entity_name = row.pop("entity_name")
+    entity = context.make("LegalEntity")
```
Comment (jbothma) [inline datasets/us/mi/med_exclusions/crawler.py:14]: Looks safe to be more specific

```suggestion
    entity = context.make("Company")
```

**Review by jbothma — CHANGES_REQUESTED** [review]
Thanks! Mainly missing some entities

**Comment by bgmello** [comment]
There are some dates I couldn't parse, I think they are an error but I'm not sure what to do in that case, do I just leave the warning?
Screenshot 2024-09-12 at 15 17 35

**Comment by jbothma** [comment]
good question.
The next step is to add a `type.date` lookup in the metadata. Any values you feel you can accurately parse yourself, you can make the `value` the date, e.g. `2023-06-27`. The rest, just make `value: null`

https://zavod.opensanctions.org/best_practices/datapatch_lookups/

**Comment by bgmello** [comment]
I added the lookup but it's still showing the warnings for the non-null values.

**Pull request merged**